db/migrate/20150114194348_create_questions.rb

```diff
@@ -0,0 +1,12 @@
+class CreateQuestions < ActiveRecord::Migration
+  def change
+    create_table :questions do |t|
+      t.string :title
+      t.text :content
+      t.datetime :posted_at
+      t.belongs_to :user
```

Comment:
when you create a migration use this:
```
t.references :user, index: true
```
if you've already distributed the migration use this:
```
add_index :questions, :user_id
```